bigchaindb/web/server.py

```diff
@@ -79,8 +80,8 @@ def create_server(settings):
     if not settings.get('threads'):
         settings['threads'] = (multiprocessing.cpu_count() * 2) + 1
 
-    debug = settings.pop('debug', False)
-    app = create_app(debug)
+    app = create_app(settings)
+    settings.pop('debug', False)
```

Comment:
I wonder why popping the value it, with a default, but not assigning it. Would simply deleting `debug` do the work?

```python
try:
    del settings['debug']
except KeyError:
    pass
```